### DISCUSSION TITLE
doc: missing TOC for aws

### DISCUSSION THREAD

**Discussion by DiscreteTom** [serverless/serverless#11041] [General]
![image](https://user-images.githubusercontent.com/28821990/167057536-ab910773-31aa-47ed-97d7-5f4463af4f00.png)

**Comment by DiscreteTom** [comment]
Sorry, didn't see `All the Serverless Framework documentation applies to AWS by default.`.

**Reply by DiscreteTom** [reply]
But this is truly inconvenient... I still think there should be a TOC and some sub-documentations to clarify AWS-specific configurations like lambda events...

**Reply by mnapoli** [reply]
Hi @DiscreteTom, thanks for the feedback.

I'm interested to learn how it is inconvenient (we recently changed the menu, so it's open for feedback).

All AWS events are still listed in the menu, we just tried to make them easier to find.

**Reply by DiscreteTom** [reply]
Hi @mnapoli , thanks for the reply.

With the previous version of this menu, we use `Cloud Providers -> XXX -> Events` to locate cloud-specific docs since we are working on a multi-cloud project. So with the newer menu, we suddenly can't find those docs haha... Yeah I see them in the `Usage` part now.

Maybe there could be a link or guidance from `Cloud Providers/AWS` to `Usage`?

**Reply by mnapoli** [reply]
I hear you, but except pointing to the left menu I'm not sure what to change here

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/720328/168833320-f9f760d4-14aa-4cbe-9763-c89ed1a5fd50.png">
